app/src/main/java/com/fstyle/androidtrainning/screen/main/ViewPagerAdapter.java

```diff
@@ -1,10 +1,10 @@
-package com.fstyle.androidtrainning.screen;
+package com.fstyle.androidtrainning.screen.main;
 
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentPagerAdapter;
 
-import com.fstyle.androidtrainning.MainActivity;
+import com.fstyle.androidtrainning.screen.main.MainActivity;
```

Comment:
ERROR: Redundant import from the same package - com.fstyle.androidtrainning.screen.main.MainActivity.